src/Workspaces/Core/Portable/Workspace/Solution/SolutionInfo.cs

```diff
@@ -42,11 +43,22 @@ public sealed class SolutionInfo
     /// </summary>
     public IReadOnlyList<AnalyzerReference> AnalyzerReferences { get; }
 
-    private SolutionInfo(SolutionAttributes attributes, IReadOnlyList<ProjectInfo> projects, IReadOnlyList<AnalyzerReference> analyzerReferences)
+    /// <summary>
+    /// Per-language analyzer config options that are used as a fallback if the option is not present in <see cref="AnalyzerConfigOptionsResult"/> produced by the compiler.
+    /// Implements a top-level (but not global) virtual editorconfig file that's in scope for all source files of the solution.
```

Comment:
what's the distinction betwene "top-level" and "global"?  wasn't sure what thsi sentence meant.

Reply:
"global" editorconfig applies to all files of the project regardless of their paths: https://github.com/dotnet/roslyn/issues/47707
The fallback options are more global then the per-project global editor config. They apply to everything in the solution (if no other editorconfig file specifies the option, we fall back).